### PR TITLE
fixing read time estimate

### PR DESCRIPTION
**Pull request by martinraison**
@yingjieMiao @eishay 
with tests :sunglasses:

### CODE REVIEW

File: kifi-backend/modules/eliza/test/com/keepit/eliza/commanders/WordCountCommanderTest.scala [hunk 1]
```diff
@@ -66,5 +66,11 @@ class WordCountCommanderTest extends Specification with ApplicationInjector{
         Await.result(wcCommander.getWordCount(uids(1), url = "http://singleWord.com"), Duration(1, SECONDS)) === 2
       }
     }
+
+    "get read time estimate" in {
+      val wordCounts = Seq(-3, -1, 0, 1, 2, 34, 123, 250, 300, 500, 5345, 15000, 15001, 10000000)
+      val readTimes = Seq(None, None, None, Some(1), Some(1), Some(1), Some(1), Some(1), Some(2), Some(2), Some(30), Some(60), Some(60), Some(60))
+      (wordCounts map WordCountCommander.wordCountToReadTimeMinutes) === readTimes
+    }
```
Comment (yingjieMiao) [inline kifi-backend/modules/eliza/test/com/keepit/eliza/commanders/WordCountCommanderTest.scala:74]: nice!

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala [hunk 2]
```diff
@@ -63,13 +59,16 @@ class WordCountCommanderImpl @Inject()(
...
+}
+
+object WordCountCommander {
+  private val WORDS_PER_MINUTE = 250
+  private val READ_TIMES = Seq(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 60)
+
+  def wordCountToReadTimeMinutes(wc: Int): Option[Int] = {
+    if (wc <= 0) None
+    else Some(READ_TIMES.dropWhile(_ * WORDS_PER_MINUTE < wc).headOption.getOrElse(READ_TIMES.last))
   }
```
Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:74]: READ_TIMES.last  is O(n)? 
also (_ \* WORDS_PER_MINUTE) is computed for a few extra times

File: kifi-backend/modules/eliza/test/com/keepit/eliza/commanders/WordCountCommanderTest.scala (hunk 1 above)
Comment (eishay) [inline kifi-backend/modules/eliza/test/com/keepit/eliza/commanders/WordCountCommanderTest.scala:74]: :green_heart:

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala (hunk 2 above)
Comment (martinraison) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:74]: I was more concerned about readability than performance, but you're point is right.

**Pull request merged**